package.json

```diff
@@ -28,7 +28,7 @@
     "iflow-lodash": "^1.1.24",
     "iflow-react-router": "^1.2.1",
     "jest": "^19.0.2",
-    "lerna": "^2.0.0-beta.38",
+    "lerna": "2.0.0-beta.38",
```

Comment:
Since we're gonna be using the local version of lerna we should depend on the exact version